modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -262,20 +262,26 @@ def get_completed_phase_number_from_phase_details(details)
 
         def get_bgs_phase_completed_dates(data)
           lc_status_array =
-            [data&.dig(:benefit_claim_details_dto, :bnft_claim_lc_status)].flatten.compact
-          return {} if lc_status_array.first.nil?
+            [data&.dig(:benefit_claim_details_dto, :bnft_claim_lc_status)].flatten&.compact
+          return {} if lc_status_array&.first&.nil?
 
-          max_completed_phase = lc_status_array.first[:phase_type_change_ind].split('').last
-          return {} if max_completed_phase.downcase.eql?('n')
+          max_completed_phase = lc_status_array&.first&.[](:phase_type_change_ind)&.split('')&.last
+          return {} if max_completed_phase&.downcase.eql?('n')
```

Comment:
we can emulate this with the example we received from VA.gov - TLDR below:
```
data = {:benefit_claim_details_dto=>{:bnft_claim_lc_status=>{:phase_type=>"Unknown"}}}
lc_status_array = [data&.dig(:benefit_claim_details_dto, :bnft_claim_lc_status)].flatten&.compact

lc_status_array
 => [{:phase_type=>"Unknown"}]

lc_status_array.first.nil?
 => false

max_completed_phase = lc_status_array.first[:phase_type_change_ind].split('').last
(irb):34:in `<main>': undefined method `split' for nil:NilClass (NoMethodError)
```

if we add :268 ( safe navigation ) I think we return out (`{}`) at :269, so add an `||` there if that's the expectation. Then we wouldn't need any of the change in :272 - :284

I'm not sure how we're just now seeing these `{:phase_type=>"Unknown"}` and maybe that's the better thing to understand?